### ISSUE TITLE
Checkmark off-center and jittery at non-default sizes

### ISSUE THREAD

**Issue by mgiuffrida** [PolymerElements/paper-checkbox#139]
Demo: https://jsfiddle.net/wzse6m00/2/

When `paper-checkbox`es are sized other than the default 18px, the checkmarks may be off-center. When a `paper-checkbox`'s container is resized (e.g. when resizing the page), the checkmark jumps around, stretches, and shrinks, making it extremely jittery.

Tested with Chrome 52 and 53 and Edge.

May be related to the root cause of #130, but neither c656a41 nor #129 fix this issue.

**Comment by bicknellr** [comment]
Yeah, this is the same problem; box dimensions for the checkmark are getting aliased to device pixels.
